Rick&Morty/RickAndMorty/Scenes/Locations/Location List/LocationsListStore.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  LocationsListStore.swift
+//  RickAndMorty
+//
+//  Created by Martin Vidovic on 05.09.2022.
+//  Copyright © 2022 STRV. All rights reserved.
+//
+
+import Foundation
+
+@MainActor final class LocationsListStore: ObservableObject {
```

Comment:
Theoretically, since you use this in UIKit, you don't need to conform to `ObservableObject`